main/user_interface.py

```diff
@@ -2,7 +2,7 @@
 import database as db
 
 
-def user_interface():
+def user_interface() -> object:
```

Comment:
We don't return any data from the user_interface module, so we do not need user_interface() -> object: